openshift/f8notification.secrets.yaml

```diff
@@ -19,4 +19,6 @@ objects:
   data:
     wit.api: https://api.prod-preview.openshift.io
     keycloak.url: https://sso.prod-preview.openshift.io
-    keycloak.realm: fabric8-test
\ No newline at end of file
+    keycloak.realm: fabric8-test
+    service.account.id: 4c83ca2d-6dcc-41c9-ac7d-b068ad4d17c5
```

Comment:
service.account.* are secrets (at least the service.account.secret is:) ). Could you please move them to the kind: Secret section?